src/app/shared/form/builder/ds-dynamic-form-ui/models/custom-switch/custom-switch.component.ts

```diff
@@ -0,0 +1,20 @@
+import { DynamicNGBootstrapCheckboxComponent } from '@ng-dynamic-forms/ui-ng-bootstrap';
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { DynamicCustomSwitchModel } from './custom-switch.model';
+
+@Component({
+  selector: 'ds-custom-switch',
+  styleUrls: ['./custom-switch.component.scss'],
+  templateUrl: './custom-switch.component.html',
+})
+export class CustomSwitchComponent extends DynamicNGBootstrapCheckboxComponent {
```

Comment:
Please add TypeDocs to describe this new component.  Also, I'm not seeing any specs related to this component.